lib/iex/lib/iex/info.ex

```diff
@@ -51,16 +51,15 @@ defimpl IEx.Info, for: Atom do
         ""
       end
 
-    mod_info = mod.module_info()
```

Comment:
@wasnotrice it seems your formatter is out of date. Please get latest master and run `make clean compile` to get the latest formatter. This file has already been formatted, please try a new one. :)

Reply:
Thank you!

Reply:
@josevalim apologies, I'll try again!